api/include/vehicles/kia_niro.h

```diff
@@ -0,0 +1,460 @@
+/**
+ * @file kia_niro.h
+ * @brief Kia Niro specific macros.
+ *
+ */
+
+
+#ifndef _KIA_NIRO_PLATFORM_INFO_H_
+#define _KIA_NIRO_PLATFORM_INFO_H_
+
+
+#include <stdint.h>
+
+
+// ********************************************************************
+//
+// WARNING
+//
+// The values listed here are carefully tested to ensure that the vehicle's
+// components are not actuated outside of the range of what they can handle.
+// By changing any of these values you risk attempting to actuate outside of the
+// vehicle's valid range. This can cause damage to the hardware and/or a
+// vehicle fault. Clearing this fault state requires additional tools.
+//
+// It is NOT recommended to modify any of these values without expert knowledge.
+//
+// ************************************************************************
+
+
+// ****************************************************************************
+// OBD MESSAGES
+// ****************************************************************************
+
+/*
+ * @brief ID of the Kia Niro's OBD steering wheel angle CAN frame.
+ *
+ */
+#define KIA_SOUL_OBD_STEERING_WHEEL_ANGLE_CAN_ID ( 0x2B0 )
+
+/*
+ * @brief ID of the Kia Niro's OBD wheel speed CAN frame.
+ *
+ */
+#define KIA_SOUL_OBD_WHEEL_SPEED_CAN_ID ( 0x386 )
+
+/*
+ * @brief ID of the Kia Niro's OBD brake pressure CAN frame.
+ *
+ */
+#define KIA_SOUL_OBD_BRAKE_PRESSURE_CAN_ID ( 0x220 )
+
+/*
+ * @brief ID of the Kia Niro's OBD speed CAN frame.
+ *
+ */
+#define KIA_SOUL_OBD_SPEED_CAN_ID ( 0x371 )
+
+/*
+ * @brief Factor to scale OBD steering angle to degrees
+ *
+ */
+#define KIA_SOUL_OBD_STEERING_ANGLE_SCALAR ( 0.1 )
+
+/**
+ * @brief Steering wheel angle message data.
+ *
+ */
+typedef struct
+{
+    int16_t steering_wheel_angle; /* 1/10 degrees */
+
+    uint8_t reserved[6]; /* Reserved. */
+} kia_soul_obd_steering_wheel_angle_data_s;
+
+/**
+ * @brief Wheel speed message data.
+ *
+ */
+typedef struct
+{
+    uint8_t wheel_speed_front_left; /* 1/4 kph */
```

Comment:
These are either 9 or 10 bit values. Can we live with this not being correct? Would anyone notice?